### PR TITLE
data-types first draft

### PR DESCRIPTION
**Pull request by rosibaj**
adding data type documentation

### CODE REVIEW

File: docs/analysis-workflows/data-types.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
+---
+id: data-types
+title: Data Types Overview
+sidebar_label: Data Types and File Formats
+---
+
+ICGC ARGO provides access to harmonized data from new programs
```
Comment (kcullion) [inline docs/analysis-workflows/data-types.md:7]: add a period.

Also, did Christina still want to stay away from the word "harmonized"?

File: docs/analysis-workflows/data-types.md [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+
+## Reads
+
+### Submitted Reads
+
+Raw reads submitted by ARGO member programs that are used as inputs to the ARGO analytic pipeline(s). Data is accepted in FASTQ or BAM format and must be compressed.
```
Comment (kcullion) [inline docs/analysis-workflows/data-types.md:13]: I think this needs rewording..

`Raw reads submitted by ARGO member programs are used as inputs within the ARGO analytic pipeline.`

File: docs/analysis-workflows/data-types.md [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+
+| Filename Pattern        | Description                             | Analysis Type    | Data Category    | Generating Workflow(s) |
+| ----------------------- | --------------------------------------- | ---------------- | ---------------- | ---------------------- |
+| \*.fq.gz, \*.fastq.gz   | FASTQ file format compressed with gzip  | sequencing_reads | Sequencing Reads | N/A                    |
+| \*.fq.bz2, \*.fastq.bz2 | FASTQ file format compressed with bzip2 | sequencing_reads | Sequencing Reads | N/A                    |
+| \*.bam                  | Binary Alignment Map (BAM) file format  | sequencing_reads | Sequencing Reads | N/A                    |
+
+### Aligned Reads
+
+A read is a sequence obtained from a single sequencing experiment. An aligned read, is a sequence that has been aligned to a common reference genome. Typically these reads can number from the hundreds of thousands to tens of millions. Aligned reads are generated internally by the ARGO alignment workflow(s) and maintained in CRAM (a compressed version of BAM that stores only reads different from the reference sequence) format.
```
Comment (kcullion) [inline docs/analysis-workflows/data-types.md:25]: remove comma here

File: docs/analysis-workflows/data-types.md [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+
+#### File Types
+
+| Filename Pattern | Description                               | Analysis Type    | Data Category    | Generating Workflow(s) |
+| ---------------- | ----------------------------------------- | ---------------- | ---------------- | ---------------------- |
+| \*.aln.cram      | CRAM file format and compressed with gzip | sequencing_reads | Sequencing Reads | DNA Seq Alignment      |
+
+### Aligned Reads Index
+
+Secondary files that are external index files for CRAM format files. CRAI files follow a naming convention to correspond to their matching CRAM file of the orginal filename suffixed with `.cram.crai`. Index files are required for selective access to genomic data inside a CRAM via CRAM slicing.
```
Comment (kcullion) [inline docs/analysis-workflows/data-types.md:35]: - I see your sentence structure now.. if you reword the above, could word this as
`These are secondary files that are external index files for CRAM format files.`

- CRAI files follow a naming convention `that` correspond to their matching CRAM file of the `original` filename suffixed with..

(not intended to emphasize these word - just highlighting what to change.

File: docs/analysis-workflows/data-types.md [hunk 5]
```diff
@@ -0,0 +1,176 @@
...
+
+#### File Types
+
+| Filename Pattern | Description                                                 | Analysis Type    | Data Category    | Generating Workflow(s) |
+| ---------------- | ----------------------------------------------------------- | ---------------- | ---------------- | ---------------------- |
+| \*.aln.cram.crai | Cram Index file format. Requires a corresponding CRAM file. | sequencing_reads | Sequencing Reads | DNA Seq Alignment      |
```
Comment (kcullion) [inline docs/analysis-workflows/data-types.md:41]: - Cram should be uppercase

File: docs/analysis-workflows/data-types.md [hunk 6]
```diff
@@ -0,0 +1,176 @@
...
+
+## Quality Control Metrics
+
+Quality control metrics are collected and recorded at several checkpoints in the ARGO Analytics pipeline to ensure that only high-quality data is released. All QC Metrics are associated to a corresponding analysis/file set that they annotate.
```
Comment (kcullion) [inline docs/analysis-workflows/data-types.md:45]: should ARGO Analytics pipeline be `ARGO Analysis pipeline` - that's what the menu item is for the docs site

File: docs/analysis-workflows/data-types.md (hunk 3 above)
Comment (rosibaj) [reply docs/analysis-workflows/data-types.md:25]: i wondered about linking, but will it be too much linking onthe page? to linke everytime we say analytics workflows?

Comment (kcullion) [reply docs/analysis-workflows/data-types.md:25]: I think at least one link is helpful somewhere on the page so they know these terms are defined elsewhere

File: docs/analysis-workflows/data-types.md [hunk 7]
```diff
@@ -0,0 +1,176 @@
...
+
+### Read Group QC
+
+file containing read group (lane) level QC metrics including --,--?.
```
Comment (kcullion) [inline docs/analysis-workflows/data-types.md:49]: I assume this sentence isn't finished, but capitalize the beginning: 

This is the file containing read group (lane) level QC metrics including --,--?.

Comment (rosibaj) [reply docs/analysis-workflows/data-types.md:49]: yes i want to discuss that with junjun (i don't know what to put here)

File: docs/analysis-workflows/data-types.md [hunk 8]
```diff
@@ -0,0 +1,176 @@
...
+Files containing quality control metrics for aligned CRAM files.
+
+#### File Types
+
+| Filename Pattern   | Description                             | Analysis Type | Data Category           | Generating Workflow(s)                                 |
+| ------------------ | --------------------------------------- | ------------- | ----------------------- | ------------------------------------------------------ |
+| \*.qc_metrics.tgz  | Generated by Samtools stats             | qc_metrics    | Quality Control Metrics | DNA Seq Alignment                                      |
+| \*.bas_metrics.tgz | Generated by Sanger bam_stats.pl script | qc_metrics    | Quality Control Metrics | Sanger WGS Variant Calling, Sanger WXS Variant Calling |
+
+### Duplicates Metrics
```
Comment (kcullion) [inline docs/analysis-workflows/data-types.md:68]: should Duplicates be plural?

File: docs/analysis-workflows/data-types.md [hunk 9]
```diff
@@ -0,0 +1,176 @@
...
+
+Data file containing duplicates metrics for aligned CRAM files. Multiple reads that match at the same position in the genome are located and tagged duplicate reads in the CRAM file, where duplicate reads are defined as originating from a single fragment of DNA.
```
Comment (kcullion) [inline docs/analysis-workflows/data-types.md:70]: if you change above introductions as above, could reword this to
`This is the data file containing duplicates metrics for aligned CRAM files.`

File: docs/analysis-workflows/data-types.md (hunk 3 above)
Comment (rosibaj) [reply docs/analysis-workflows/data-types.md]: what if i expand the intro paragraph and put it there?

Comment (kcullion) [reply docs/analysis-workflows/data-types.md]: yeah that works!

File: docs/analysis-workflows/data-types.md [hunk 10]
```diff
@@ -0,0 +1,176 @@
...
+
+#### File Types
+
+| Filename Pattern          | Description                                | Analysis Type | Data Category           | Generating Workflow(s) |
+| ------------------------- | ------------------------------------------ | ------------- | ----------------------- | ---------------------- |
+| \*.duplicates_metrics.tgz | Generated by biobambam2 bammarkduplicates2 | qc_metrics    | Quality Control Metrics | DNA Seq Alignment      |
+
+### OxoG Metrics
+
+Data file containing OxoG metrics, metrics quantifying the error rate resulting from oxidative artifacts, for aligned CRAM files.
```
Comment (kcullion) [inline docs/analysis-workflows/data-types.md:80]: not sure, as above...seems like something is missing

Data file containing OxoG metrics, i.e. metrics quantifying the error rate resulting from oxidative artifacts for aligned CRAM files.

File: docs/analysis-workflows/data-types.md [hunk 11]
```diff
@@ -0,0 +1,176 @@
...
+
+### Read Group QC
+
+file containing read group (lane) level QC metrics including --,--?.
+
+#### File Types
+
+| Filename Pattern       | Description                                    | Analysis Type | Data Category           | Generating Workflow(s) |
+| ---------------------- | ---------------------------------------------- | ------------- | ----------------------- | ---------------------- |
+| \*.ubam_qc_metrics.tgz | Generated by Picard CollectQualityYieldMetrics | qc_metrics    | Quality Control Metrics | DNA Seq Alignment      |
```
Comment (kcullion) [inline docs/analysis-workflows/data-types.md:55]: I like that on this page https://docs.icgc-argo.org/docs/analysis-workflows/dna-pipeline
there are links out to define somethings (eg. https://gatk.broadinstitute.org/hc/en-us/articles/360042475912-CollectQualityYieldMetrics-Picard-)
Should we do that on this page too?

File: docs/analysis-workflows/data-types.md [hunk 12]
```diff
@@ -0,0 +1,176 @@
...
+
+#### File Types
+
+| Filename Pattern    | Description                          | Analysis Type | Data Category           | Generating Workflow(s) |
+| ------------------- | ------------------------------------ | ------------- | ----------------------- | ---------------------- |
+| \*.oxog_metrics.tgz | Generated by GATK CollectOxoGMetrics | qc_metrics    | Quality Control Metrics | DNA Seq Alignment      |
+
+### Ploidy and Purity Estimation
+
+Data file containing purity and ploidy estimate.
```
Comment (kcullion) [inline docs/analysis-workflows/data-types.md:90]: is there anything else to say to define this?

File: docs/analysis-workflows/data-types.md [hunk 13]
```diff
@@ -0,0 +1,176 @@
...
+
+#### File Types
+
+| Filename Pattern     | Description                          | Analysis Type | Data Category           | Generating Workflow(s)     |
+| -------------------- | ------------------------------------ | ------------- | ----------------------- | -------------------------- |
+| \*.ascat_metrics.tgz | Estimated by Sanger ASCAT CNV caller | qc_metrics    | Quality Control Metrics | Sanger WGS Variant Calling |
+
+### Cross Sample Contamination
+
+Data file containing cross sample contamination estimate.
```
Comment (kcullion) [inline docs/analysis-workflows/data-types.md:100]: is there anything else to say to define this?

File: docs/analysis-workflows/data-types.md [hunk 14]
```diff
@@ -0,0 +1,176 @@
...
+
+#### File Types
+
+| Filename Pattern             | Description                                                                                                                                     | Analysis Type | Data Category           | Generating Workflow(s)     |
+| ---------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------- | ------------- | ----------------------- | -------------------------- |
+| \*.contamination_metrics.tgz | Estimated by Sanger `verifyBamHomChk.pl` script, which provides information to determine whether the sample is possibly contaminated or swapped | qc_metrics    | Quality Control Metrics | Sanger WGS Variant Calling |
+
+### Genotyped Gender Metrics
+
+Data file containing genotypes comparison results of CRAM files from the same donor, including the fraction of matched genotypes and inferred the donor gender.
```
Comment (kcullion) [inline docs/analysis-workflows/data-types.md:110]: not sure if this is worded correctly?should it be

`Data file containing genotypes comparison results of CRAM files from the same donor, including the fraction of matched genotypes that infers the gender of the donor.`

File: docs/analysis-workflows/data-types.md [hunk 15]
```diff
@@ -0,0 +1,176 @@
...
+
+#### File Types
+
+| Filename Pattern                | Description                                          | Analysis Type | Data Category           | Generating Workflow(s)     |
+| ------------------------------- | ---------------------------------------------------- | ------------- | ----------------------- | -------------------------- |
+| \*.genotyped_gender_metrics.tgz | Generated by Sanger `compareBamGenotypes.pl` script. | qc_metrics    | Quality Control Metrics | Sanger WGS Variant Calling |
+
+### Variant Calling Supplements
+
+Data file containing variation results, temporary files or runtime metrics information generated by variant callers used in the ARGO analytic pipeline.
```
Comment (kcullion) [inline docs/analysis-workflows/data-types.md:120]: - [ ] Data file containing variation results. These are temporary files or runtime metrics information generated by variant callers used in the ARGO Analysis pipeline.

File: docs/analysis-workflows/data-types.md (hunk 13 above)
Comment (rosibaj) [reply docs/analysis-workflows/data-types.md:100]: need to talk tok bioinfo

File: docs/analysis-workflows/data-types.md (hunk 12 above)
Comment (rosibaj) [reply docs/analysis-workflows/data-types.md:90]: need to talk to bioinfo

File: docs/analysis-workflows/data-types.md (hunk 11 above)
Comment (rosibaj) [reply docs/analysis-workflows/data-types.md:55]: probably, but im a but worried about scalability on updates. if its on both pages then its double the links to update...

Comment (kcullion) [reply docs/analysis-workflows/data-types.md:55]: yeah. Which page is best to have these definitions? Maybe this definition page?

File: docs/analysis-workflows/data-types.md [hunk 16]
```diff
@@ -0,0 +1,176 @@
...
+
+#### File Types
+
+| Filename Pattern | Description     | Analysis Type   | Data Category         | Generating Workflow(s)     |
+| ---------------- | --------------- | --------------- | --------------------- | -------------------------- |
+| \*.cnv.vcf.gz    | VCF file format | variant_calling | Copy Number Variation | Sanger WGS Variant Calling |
+
+### Raw InDel Calls
+
+Data files containing the simple insertions and deletions (indels) data called from aligned reads, which have not yet been annotated but do have some filtering flags added.
```
Comment (kcullion) [inline docs/analysis-workflows/data-types.md:166]: in some you say "filtering flags added" - do people know what this means?

File: docs/analysis-workflows/data-types.md (hunk 14 above)
Comment (rosibaj) [reply docs/analysis-workflows/data-types.md:110]: not quite but  i see the issue and fixed it

File: docs/analysis-workflows/data-types.md (hunk 15 above)
Comment (rosibaj) [reply docs/analysis-workflows/data-types.md:120]: i fixed the commas rather than a (.)

File: docs/analysis-workflows/data-types.md [hunk 17]
```diff
@@ -0,0 +1,176 @@
...
+
+#### File Types
+
+| Filename Pattern | Description     | Analysis Type   | Data Category               | Generating Workflow(s)                                 |
+| ---------------- | --------------- | --------------- | --------------------------- | ------------------------------------------------------ |
+| \*.indel.vcf.gz  | VCF file format | variant_calling | Simple Nucleotide Variation | Sanger WGS Variant Calling, Sanger WXS Variant Calling |
+
+### VCF Index
+
+Secondary files that are external index files for VCF format files. TBI files follow a naming convention to correspond to their matching VCF file of the original filename suffixed with `.vcf.gz.tbi`. Index files are required for selective access to genomic data inside a VCF file.
```
Comment (kcullion) [inline docs/analysis-workflows/data-types.md:176]: not sure if this page is done yet...just see some other metrics file on the diagrams on https://docs.icgc-argo.org/docs/analysis-workflows/dna-pipeline 

like Timing metrics supplement, Normal alignment QC etc,  will those be added?

File: docs/analysis-workflows/data-types.md (hunk 15 above)
Comment (kcullion) [reply docs/analysis-workflows/data-types.md:120]: also changing ARGO analytic pipeline to ARGO Analysis pipeline like in the menu item label

File: docs/analysis-workflows/data-types.md (hunk 17 above)
Comment (rosibaj) [reply docs/analysis-workflows/data-types.md:176]: 1) is in variant calling supplement 2)  is Alignment QC (coud be for normal or otherwise)

File: docs/analysis-workflows/data-types.md (hunk 9 above)
Comment (rosibaj) [reply docs/analysis-workflows/data-types.md:70]: no i there where is correct here

File: docs/analysis-workflows/data-types.md [hunk 18]
```diff
@@ -0,0 +1,176 @@
...
+
+Data file containing cross sample contamination estimate.
+
+#### File Types
+
+| Filename Pattern             | Description                                                                                                                                     | Analysis Type | Data Category           | Generating Workflow(s)     |
+| ---------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------- | ------------- | ----------------------- | -------------------------- |
+| \*.contamination_metrics.tgz | Estimated by Sanger `verifyBamHomChk.pl` script, which provides information to determine whether the sample is possibly contaminated or swapped | qc_metrics    | Quality Control Metrics | Sanger WGS Variant Calling |
+
+### Genotyped Gender Metrics
```
Comment (junjun-zhang) [inline docs/analysis-workflows/data-types.md:108]: should this be `Genotyping Inferred Gender`

File: docs/analysis-workflows/data-types.md [hunk 19]
```diff
@@ -0,0 +1,176 @@
...
+
+Data file containing genotypes comparison results of CRAM files from the same donor, including the fraction of matched genotypes and inferred donor gender.
+
+#### File Types
+
+| Filename Pattern                | Description                                          | Analysis Type | Data Category           | Generating Workflow(s)     |
+| ------------------------------- | ---------------------------------------------------- | ------------- | ----------------------- | -------------------------- |
+| \*.genotyped_gender_metrics.tgz | Generated by Sanger `compareBamGenotypes.pl` script. | qc_metrics    | Quality Control Metrics | Sanger WGS Variant Calling |
+
+### Variant Calling Supplements
```
Comment (junjun-zhang) [inline docs/analysis-workflows/data-types.md:118]: `Variant Calling Supplement` is a dataType under different variant call types, it contains supplementary information for variant calling, should it be under `Variant Calls` section not `Quality Control Metrics`?

File: docs/analysis-workflows/data-types.md (hunk 1 above)
Comment (rosibaj) [reply docs/analysis-workflows/data-types.md]: did you remember what word to use instead?

Comment (rosibaj) [reply docs/analysis-workflows/data-types.md]: changed to: ICGC ARGO provides access to data from ARGO member programs generated through the standardized [ARGO Analysis pipelines](/docs/analysis_workflows/data-types).

File: docs/analysis-workflows/data-types.md (hunk 11 above)
Comment (rosibaj) [reply docs/analysis-workflows/data-types.md:55]: discussed with chrstina - link workflow outputs to this page for each document.

File: docs/analysis-workflows/data-types.md (hunk 1 above)
Comment (kcullion) [reply docs/analysis-workflows/data-types.md]: yeah that sounds good...
I think it was just using "analyzed" instead...I'd have to look in my notes

File: docs/analysis-workflows/data-types.md (hunk 18 above)
Comment (rosibaj) [reply docs/analysis-workflows/data-types.md:108]: Genotyped Gender Metrics is the datatype of the file, so I went with that for consistency.

Comment (junjun-zhang) [reply docs/analysis-workflows/data-types.md:108]: `Genotyping Inferred Gender` is the dataType, see here: https://song.rdpc.cancercollaboratory.org/studies/OCCAMS-GB/analysis

Comment (rosibaj) [reply docs/analysis-workflows/data-types.md:108]: oh i see - i was going off of this: https://wiki.oicr.on.ca/display/icgcargotech/File+dataType+specification

File: docs/analysis-workflows/data-types.md (hunk 19 above)
Comment (rosibaj) [reply docs/analysis-workflows/data-types.md:118]: yes this makes sense! i have moved it.

File: docs/analysis-workflows/data-types.md (hunk 18 above)
Comment (rosibaj) [reply docs/analysis-workflows/data-types.md:108]: changed.

File: docs/analysis-workflows/data-types.md (hunk 10 above)
Comment (rosibaj) [reply docs/analysis-workflows/data-types.md]: rearranged to 2 sentences

File: docs/analysis-workflows/data-types.md (hunk 13 above)
Comment (rosibaj) [reply docs/analysis-workflows/data-types.md]: confirmed should be enough with christina

File: docs/data/quality-metrics.md [hunk 20]
```diff
@@ -0,0 +1,80 @@
+---
+id: qc-metrics
+title: Quality Control Metrics
+sidebar_label: QC Metrics
+---
+
+ICGC ARGO provides access to data from ARGO member programs generated through the standardized [ARGO Analysis pipelines](/docs/analysis_workflows/data-types). The following page lists quality control metric analyses and data files generated by the ARGO Analysis pipelines.
```
Comment (kcullion) [inline docs/data/quality-metrics.md:7]: should it be

quality control metric `analysis` and data files

Comment (kcullion) [reply docs/data/quality-metrics.md:7]: or is this plural?

File: docs/data/reads.md [hunk 21]
```diff
@@ -0,0 +1,39 @@
...
+id: reads
+title: Reads
+sidebar_label: Reads
+---
+
+ICGC ARGO provides access to data from ARGO member programs generated through the standardized [ARGO Analysis pipelines](/docs/analysis_workflows/data-types). The following page lists genomic read analyses and data files generated by the ARGO Analysis pipelines.
+
+## Submitted Reads
+
+Raw reads submitted by ARGO member programs are used as inputs within the ARGO analytic pipeline. Data is accepted in FASTQ or BAM (aligned or unaligned) formats, must be compressed, and can be aligned or unaligned.
```
Comment (kcullion) [inline docs/data/reads.md:11]: should `aligned or unaligned` be repeated in this sentence?

File: docs/data/reads.md [hunk 22]
```diff
@@ -0,0 +1,39 @@
...
+
+| Filename Pattern        | Description                             | Analysis Type    | Data Category    | Generating Workflow(s) |
+| ----------------------- | --------------------------------------- | ---------------- | ---------------- | ---------------------- |
+| \*.fq.gz, \*.fastq.gz   | FASTQ file format compressed with gzip  | sequencing_reads | Sequencing Reads | N/A                    |
+| \*.fq.bz2, \*.fastq.bz2 | FASTQ file format compressed with bzip2 | sequencing_reads | Sequencing Reads | N/A                    |
+| \*.bam                  | Binary Alignment Map (BAM) file format  | sequencing_reads | Sequencing Reads | N/A                    |
+
+## Aligned Reads
+
+A read is a sequence obtained from a single sequencing experiment. An aligned read is a sequence that has been aligned to a common reference genome. Typically these reads can number from the hundreds of thousands to tens of millions. Aligned reads are generated internally by the ARGO alignment workflow(s) and maintained in CRAM (a compressed version of BAM that stores only reads different from the reference sequence) format. ARGO reads are aligned to the GRCh38 Human Reference Genome.
```
Comment (kcullion) [inline docs/data/reads.md:23]: CRAM is defined here...is this the first instance of it? Like should it be defined when used on the other page too?

File: docs/data/reads.md (hunk 21 above)
Comment (kcullion) [reply docs/data/reads.md:11]: also..should "unaligned reads" be a section on this page too? or does it just not produce files?

File: docs/data/variant-calls.md [hunk 23]
```diff
@@ -0,0 +1,71 @@
...
+id: variant-calls
+title: Variant Calls
+sidebar_label: Variant Calls
+---
+
+ICGC ARGO provides access to data from ARGO member programs generated through the standardized [ARGO Analysis pipelines](/docs/analysis_workflows/analysis-overview). The following page lists variant calling analyses and data files generated by the ARGO Analysis pipelines.
+
+## Raw SNV Calls
+
+Data files containing Single Nucleotide Variations (SNV) called from aligned reads, which have not yet been annotated but do have some filtering flags added.
```
Comment (kcullion) [inline docs/data/variant-calls.md:11]: does the user know what "filtering flags" are? asking for a friend haha

File: docs/data/reads.md (hunk 21 above)
Comment (rosibaj) [reply docs/data/reads.md:11]: no unaligned reads are what are submitted to us. ARGO does not produce nor host unaligned reads.

Comment (rosibaj) [reply docs/data/reads.md:11]: um the () is specifically that the BAM can be wither, where as FASTQ is only unaligned.

Comment (kcullion) [reply docs/data/reads.md:11]: this just seems redundant, but maybe I'm reading it wrong?

Data is accepted in FASTQ or BAM (`aligned or unaligned`) formats, must be compressed, and `can be aligned or unaligned`

File: docs/data/quality-metrics.md (hunk 20 above)
Comment (rosibaj) [reply docs/data/quality-metrics.md:7]: its plural

File: docs/data/reads.md (hunk 22 above)
Comment (rosibaj) [reply docs/data/reads.md:23]: yes, originally this was all on one page. this is a good question... hmm i lean towards leaving the definition here as the other pages can reference this?

Comment (kcullion) [reply docs/data/reads.md:23]: sure..or both pages if you want

File: docs/analysis-workflows/data-types.md (hunk 12 above)
Comment (lindaxiang) [reply docs/analysis-workflows/data-types.md]: Tumour purity measures the fraction of cancer cells in a tumour sample.
Tumour ploidy measures the number of sets of chromosomes in a tumour cell.

File: docs/data/quality-metrics.md [hunk 24]
```diff
@@ -0,0 +1,80 @@
...
+## Alignment QC
+
+Files containing quality control metrics for aligned CRAM files.
+
+### File Types
+
+| Filename Pattern   | Description                             | Analysis Type | Data Category           | Generating Workflow(s)                                 |
+| ------------------ | --------------------------------------- | ------------- | ----------------------- | ------------------------------------------------------ |
+| \*.qc_metrics.tgz  | Generated by Samtools stats             | qc_metrics    | Quality Control Metrics | DNA Seq Alignment                                      |
+| \*.bas_metrics.tgz | Generated by Sanger bam_stats.pl script | qc_metrics    | Quality Control Metrics | Sanger WGS Variant Calling, Sanger WXS Variant Calling |
```
Comment (lindaxiang) [inline docs/data/quality-metrics.md:30]: @rosibaj , a minor comment. do you think it is easy to read by separating different workflows with `enter`?

File: docs/data/quality-metrics.md [hunk 25]
```diff
@@ -0,0 +1,80 @@
...
+
+### File Types
+
+| Filename Pattern    | Description                          | Analysis Type | Data Category           | Generating Workflow(s) |
+| ------------------- | ------------------------------------ | ------------- | ----------------------- | ---------------------- |
+| \*.oxog_metrics.tgz | Generated by GATK CollectOxoGMetrics | qc_metrics    | Quality Control Metrics | DNA Seq Alignment      |
+
+## Ploidy and Purity Estimation
+
+Data file containing purity and ploidy estimate.
```
Comment (lindaxiang) [inline docs/data/quality-metrics.md:54]: --> Data file containing tumour purity and ploidy estimate.

Comment (rosibaj) [reply docs/data/quality-metrics.md:54]: changed

File: docs/data/quality-metrics.md (hunk 24 above)
Comment (rosibaj) [reply docs/data/quality-metrics.md:30]: @lindaxiang hmm yess i will try and see if ther eis an accepted markdown configuration docusaurus to do this

File: docs/data/variant-calls.md (hunk 23 above)
Comment (rosibaj) [reply docs/data/variant-calls.md:11]: hahah Christina said yes lol

File: docs/data/quality-metrics.md (hunk 24 above)
Comment (rosibaj) [reply docs/data/quality-metrics.md:30]: i tried a few things, but cant get this right now. ill continue to investigate for future.

File: docs/data/reads.md (hunk 22 above)
Comment (rosibaj) [reply docs/data/reads.md:23]: hmm i think that its best just to leave it - the qc files are all past the aligned stage, so i think people already know what a CRAM is at this point.  lets add later if needed.

File: docs/data/quality-metrics.md (hunk 24 above)
Comment (lindaxiang) [reply docs/data/quality-metrics.md:30]: No worries Rosi. This is not critical at all. We can get back to it anytime later if needed.

File: docs/data/reads.md (hunk 21 above)
Comment (rosibaj) [reply docs/data/reads.md]: updated to `Data is accepted in FASTQ, unaligned BAM, or aligned BAM formats, and must be compressed for submissions.`

Comment (kcullion) [reply docs/data/reads.md]: yeah I think that's clearer...Could also write as

Data is accepted in FASTQ, unaligned/aligned BAM formats, and must be compressed for submissions.

**Review by kcullion — APPROVED** [review]

**Review by lindaxiang — APPROVED** [review]

File: docs/data/reads.md [hunk 26]
```diff
@@ -0,0 +1,39 @@
...
```
Comment (christinayung) [inline docs/data/reads.md:23]: Do we have a link to the GRCh38 Human Reference Genome for download?

Comment (rosibaj) [reply docs/data/reads.md:23]: Its linked here: https://docs.icgc-argo.org/docs/analysis-workflows/dna-pipeline#inputs 

I can also add here if desired.

Comment (rosibaj) [reply docs/data/reads.md:23]: added

**Review by joneubank — APPROVED** [review]
